ci/benchmark_bencher_baseline.yml

```diff
@@ -0,0 +1,62 @@
+include:
+  - local: 'ci/base.yml'
+
+.benchmark_model_stencils:
+  stage: benchmark
+  script:
+    - | # Run the benchmarks
```

Comment:
I was thinking of putting this inside a new nox session in the noxfile.py. This would make it easier for debugging in case something doesn't work in the future. In the nox tutorial there is an example with coverage, which is very similar to our use-case https://nox.thea.codes/en/stable/tutorial.html#requiring-sessions

Reply:
done